### PR TITLE
PTL-1200 add details of not null to tables basics

### PR DESCRIPTION
**Pull request by wjhendry**
also copies yesterday's Ray Chee update to Previous

Thank you for contributing to the documentation.

Do the changes you have made apply to both Current and Previous versions?
Y

Have you checked all new or changed links?
Y

Is there anything else you would like us to know?
N

For reference: 

  - We have an [internal contributions guide](https://www.notion.so/genesisglobal/Contributing-new-documentation-75953fb245f246ff872789035451a0c4)
  - We have a [style guide](https://www.notion.so/genesisglobal/Documentation-style-guide-5b04ec6fe12f4262b90d192effd8059b) 

**This week's exciting excerpts from the style guide**

- We write in UK English, with Oxford English Dictionary (OED) spellings, grammar and vocabulary.  

Use the present tense wherever possible. Only use another tense where it is strictly necessary.

- Present tense (preferred whenever possible): *The script creates a new folder.*
- Future tense (avoid whenever possible): *The script will create a new folder.*

### CODE REVIEW

File: versioned_docs/version-previous/02_database/01_fields-tables-views/01_fields/01_fields-basics.md [hunk 1]
```diff
@@ -29,23 +29,44 @@ The following field types are available:
...
-    field(name = "DESCRIPTION", type = STRING, nullable = true)
+    field("PROJECT_ID", type = STRING)
+    field("PROJECT_OPERATIONAL_STATUS", type = ENUM("OPERATIONAL", "NEW_BUILD", default = "NEW_BUILD"))
 }
 ```
-## Field type parameters
+## Field parameters
+You can add one or more parameters to any field that you declare.
 
 | Parameter | Default value | Sample Usage | Value Type | Description |
```
Comment (GuiWar) [inline versioned_docs/version-previous/02_database/01_fields-tables-views/01_fields/01_fields-basics.md:43]: Name field is missing. and I believe name and type are mandatory.

File: versioned_docs/version-previous/02_database/01_fields-tables-views/01_fields/01_fields-basics.md [hunk 2]
```diff
@@ -29,23 +29,44 @@ The following field types are available:
...
 |---|---|---|---|---|
 | default | null | `field(name = "AGE", type = INT, default=20)` | The same type as the field | Set a default value for the field where not supplied on a database insert. Mandatory for `ENUM` |
```
Comment (GuiWar) [inline versioned_docs/version-previous/02_database/01_fields-tables-views/01_fields/01_fields-basics.md:45]: default is mandatory for ENUM inside the ENUM declaration. For example: field(name="COUNTRY", type=ENUM("BRAZIL", "KUWAIT","BHUTAN",default="BRAZIL")

File: versioned_docs/version-previous/02_database/01_fields-tables-views/01_fields/01_fields-basics.md [hunk 3]
```diff
@@ -29,23 +29,44 @@ The following field types are available:
...
-| format | N/A | `field(name = "DESCRIPTION", type = BIG_DECIMALS, format="#,##0.000")` | Dependent on type | Set the maxSize of a field. Applicable for `STRING`, `RAW`  |
-| sensitive | false | `field(name = "PASSWORD", type = STRING, sensitive=true)` | boolean | true if the field should be masked on its toString() output  |
-| nullable | true | `field(name = "USER_NAME", type = STRING, sensitive=true)` | boolean | true if the field should be masked on its toString() output  |
+| format | N/A | `field(name = "SETTLEMENT_AMOUNT", type = BIG_DECIMAL, format="#,##0.000")` | Dependent on type | Set the valid format of a field entry  |
+| sensitive | false | `field(name = "PASSWORD", type = STRING, sensitive=true)` | boolean | true if the field should be obfuscated in the system logs  |
+| nullable | true | `field(name = "DESCRIPTION", type = STRING, nullable=false)`; see below | boolean | false if the field is not nullable  |
```
Comment (GuiWar) [inline versioned_docs/version-previous/02_database/01_fields-tables-views/01_fields/01_fields-basics.md:49]: This argument is not available if type = ENUM

**Review by GuiWar — CHANGES_REQUESTED** [review]
Hi John. There are some changes that needs to be addressed here. Happy to huddle to sort this out.

**Comment by wjhendry** [comment]
> Hi John. There are some changes that needs to be addressed here. Happy to huddle to sort this out.

Should all be there now

**Review by dbarros1979 — APPROVED** [review]

**Review by GuiWar — APPROVED** [review]

**Pull request merged**